python.py

```diff
@@ -0,0 +1,20 @@
+# Here is a comment
```

Comment:
Typo inside comment in token 'Here'. Possible corrections:were (3),here (1),mere (0)